### ISSUE TITLE
Python/tests.py doesn't have an 'execute' flag

### ISSUE THREAD

**Issue by yurivict** [primme/primme#19]

**Comment by eromero-vlc** [comment]
Can you briefly explain why you need this. If you want just to test the python interface, just run make test on Python directory. That will run examples.py.

**Comment by yurivict** [comment]
```make test``` runs examples.py:
```
$ make test
===>  Testing for py27-primme-2.1.5.20181011
===>   py27-primme-2.1.5.20181011 depends on package: py27-nose>=0 - found
===>   py27-primme-2.1.5.20181011 depends on file: /usr/local/bin/python2.7 - found
gmake[1]: Entering directory '/usr/ports/math/py-primme/work-py27/primme-6a0fa27/Python'
python2 examples.py
[99. 98. 97.]
[96. 95. 94.]
[4. 3. 5.]
[0.79488437 0.85890809 0.87174328]
(0.01105189323425293, 194)
(0.006082057952880859, 72)
```

```tests.py``` has the python shebang, and no executable bit, and also does something other than what ```examples.py``` does. Files that have shebang should have the executable bit, otherwise how can shebang work.

**Comment by eromero-vlc** [comment]
Fixed at f372ffb71be67d69892eb3b642aa314f6a00633b
